MidasCivil_Adapter/Convert/ToBHoM/Results/ToBarStress.cs

```diff
@@ -65,6 +67,43 @@ public static BarStress ToBarStress(this List<string> delimitted, string forceUn
 
         /***************************************************/
 
+        public static BarStress ToBarStressAPI(JsonElement item)
```

Comment:
```suggestion
        public static BarStress ToBarStress(JsonElement item)
```
the function name should match the filename, also the inputs are different so you can have the old and new method. This comment applies to all the changed files in Convert/ToBHoM/Results.

Reply:
Updated where the new methods have a different input, both new and old uses List<string> for von Mises and Stresses (mesh)